connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerTask.java

```diff
@@ -70,7 +73,9 @@ public WorkerTask(ConnectorTaskId id,
                       TargetState initialState,
                       ClassLoader loader,
                       ConnectMetrics connectMetrics,
-                      RetryWithToleranceOperator retryWithToleranceOperator) {
+                      RetryWithToleranceOperator retryWithToleranceOperator,
+                      Time time,
+                      StatusBackingStore statusBackingStore) {
```

Comment:
I'm wondering if we really want to pass in the StatusBackingStore interface, or whether we should define a new interface? It'd be nice to constrain future developers to only use the methods related to active topics. It's a bit more complicated, but better captures the current intent to just allow the worker tasks to check whether a topic is considered active for a connector and if not to record that a topic has been used.

Reply:
Neither approach is terribly clean. I think the current approach certainly works, but as you point out if we add something like `TopicTracker` as a superinterface of `StatusBackingStore` then we'd be inconsistent with the pre-existing status methods.

Let's keep things simple.